FeedStoreChallenge/Infrastructure/CoreData/ManagedCache.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Andrew Marmion on 28/08/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+final class ManagedCache: NSManagedObject {
+	@NSManaged var timestamp: Date
+	@NSManaged var feed: NSOrderedSet
+}
+
+extension ManagedCache {
+	static func fetch(from context: NSManagedObjectContext) throws -> ManagedCache? {
+		let request = NSFetchRequest<ManagedCache>(entityName: entity().name!)
+		request.returnsObjectsAsFaults = false
+		return try context.fetch(request).first
+	}
+
+	static func createNewUniqueCacheInstance(in context: NSManagedObjectContext) throws -> ManagedCache {
+		try fetch(from: context).map(context.delete(_:))
```

Comment:
Suggestion:
Here `(_:)` can be omitted to make the code shorter
```suggestion
		try fetch(from: context).map(context.delete)
```

Reply:
Done